src/vs/workbench/contrib/terminal/browser/media/shellIntegration.ps1

```diff
@@ -25,23 +25,23 @@ if ($env:VSCODE_ENV_REPLACE) {
 	$Split = $env:VSCODE_ENV_REPLACE.Split(":")
 	foreach ($Item in $Split) {
 		$Inner = $Item.Split('=')
-		[Environment]::SetEnvironmentVariable($Inner[0], $Inner[1])
+		[Environment]::SetEnvironmentVariable($Inner[0], $Inner[1].Replace('\x3a', ':'))
```

Comment:
Can you clarify what this does?

Reply:
When vscode sets the variable it will ensure all values have `:` encoded as `\x3a'`, so to test this out you'll want to encode the values like this:

```
VAR1=VAL1:VAR2=http\x3a//github.com
```